src/install/oracle/linux/install-zypper.mdx

```diff
@@ -0,0 +1,16 @@
+---
+componentType: default
+headingText: Download using Zypper
```

Comment:
It's a bit confusing for there to be a "Download using X", then later "Download the integration." What exactly are they downloading here? How does that differentiate from other installs later? Maybe more context or a noun in the title